_tools/redirects/redirects.csv

```diff
@@ -38,19 +35,19 @@ source,destination
 /development/cpp/configuring_an_ide/visual_studio.html,/engine_details/development/configuring_an_ide/visual_studio.html
 /development/cpp/configuring_an_ide/visual_studio_code.html,/engine_details/development/configuring_an_ide/visual_studio_code.html
 /development/cpp/configuring_an_ide/xcode.html,/engine_details/development/configuring_an_ide/xcode.html
-/development/cpp/core_types.html,/engine_details/development/core_and_modules/core_types.html
-/development/cpp/custom_audiostreams.html,/engine_details/development/core_and_modules/custom_audiostreams.html
-/development/cpp/custom_godot_servers.html,/engine_details/development/core_and_modules/custom_godot_servers.html
-/development/cpp/custom_modules_in_cpp.html,/engine_details/development/core_and_modules/custom_modules_in_cpp.html
-/development/cpp/custom_resource_format_loaders.html,/engine_details/development/core_and_modules/custom_resource_format_loaders.html
-/development/cpp/index.html,/engine_details/development/core_and_modules/index.html
-/development/cpp/inheritance_class_tree.html,/engine_details/development/core_and_modules/inheritance_class_tree.html
-/development/cpp/introduction_to_godot_development.html,/engine_details/development/core_and_modules/introduction_to_godot_development.html
+/development/cpp/core_types.html,/engine_details/architecture/core_types.html
+/development/cpp/custom_audiostreams.html,/engine_details/architecture/custom_audiostreams.html
+/development/cpp/custom_godot_servers.html,/engine_details/architecture/custom_godot_servers.html
+/development/cpp/custom_modules_in_cpp.html,/engine_details/architecture/custom_modules_in_cpp.html
+/development/cpp/custom_resource_format_loaders.html,/engine_details/architecture/custom_resource_format_loaders.html
+/development/cpp/index.html,/engine_details/architecture/index.html
+/development/cpp/inheritance_class_tree.html,/engine_details/architecture/inheritance_class_tree.html
+/development/cpp/introduction_to_godot_development.html,/engine_details/architecture/introduction_to_godot_development.html
```

Comment:
This one isn't working; it was probably moved to `contributing.godotengine.org`.